### PR TITLE
imports openapi spec and orders left hand nav

### PR DESCRIPTION
**Pull request by stockholmux**
**Issue number:**

Closes # n/a

**Description of changes:**

Imports OpenAPI from bottlerocket-os/bottlerocket, adds styling, fixes order of left hand nav to make API push to bottom

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

File: content/en/os/1.13.x/api/endpoints/index.markdown [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title = "API Endpoints"
+type = "swagger"
+description = "Paths and methods for the API"
++++
+
+The following output is generated from [Bottlerocket's OpenAPI Spec](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/api/openapi.yaml).
+
+---
+
+{{<  swaggerui src="../../../../external/openapi.yaml" >}}
```
Comment (mchaker) [inline content/en/os/1.13.x/api/endpoints/index.markdown:11]: Not sure if this is expected or not, but when I click on e.g. `GET /settings` to expand that row, then close it (not having clicked "try it out" or any buttons), I see this error at the top of the page:

image

**Review by mchaker — APPROVED** [review]
LGTM - just a confusion/nit around values in quotes

File: content/en/os/1.13.x/api/endpoints/index.markdown (hunk 1 above)
Comment (stockholmux) [reply content/en/os/1.13.x/api/endpoints/index.markdown:11]: No, not expected.

However, this [appears to be an issue with Swagger UI](https://stackoverflow.com/questions/65584131/weird-could-not-resolve-reference-undefined-not-found-messages-in-swagger-ui). The Hugo theme just delivers it as a dependency and _it_ generates the HTML. The `$ref` appears to be emitted from the javascript used by swagger ui. 

TL;DR: we can't fix this. The best we can do is hide the on-page error, which is sad from an engineering standpoint but not really user-facing.
